src/components/ParamsBox.tsx

```diff
@@ -69,13 +74,16 @@ const ParamsBox: FunctionComponent<{}> = () => {
 						</select>
 					)}
 					{FiniteParams.is(selectedEntity.params) && (
-						<input type="text" pattern={pattern} value={selectedEntity.params.stringFormat} onChange={onChangeMagazine}/>
+						<input type="text" pattern="^(?:b|g)+$" value={selectedEntity.params.stringFormat} onChange={onChangeMagazine}/>
```

Comment:
Looks like you reverted e807beded77a5801005d2a002410c33b64c6018a

Reply:
oop. That’s an error